en/setup/fast-rtps-installation.md

```diff
@@ -0,0 +1,95 @@
+# Fast RTPS installation ![](../../assets/fastrtps/eprosima_logo.png)
+
+[_eprosima Fast RTPS_](http://eprosima-fast-rtps.readthedocs.io/en/latest/) is a C++ implementation of the RTPS (Real Time Publish Subscribe) protocol, which provides publisher-subscriber communications over unreliable transports such as UDP, as defined and maintained by the Object Management Group (OMG) consortium. RTPS is also the wire interoperability protocol defined for the Data Distribution Service (DDS) standard, again by the OMG.
+
+**NOTE**: The information here was extracted from original [_eprosima Fast RTPS_ documentation](http://eprosima-fast-rtps.readthedocs.io/en/latest/)
+
+## [Requirements](http://eprosima-fast-rtps.readthedocs.io/en/latest/requirements.html#requirements)
+
+*eProsima Fast RTPS* requires the following packages to work.
+
+### Common Dependencies
+
+#### Gtest
+
+Gtest is needed to compile the tests when building from sources.
```

Comment:
This is not clear enough. We require in our documentation installation instructions for the major platforms (e.g. apt-get install XYZ or brew install XYZ).